pynxtools/dataconverter/convert.py

```diff
@@ -75,6 +75,8 @@ def get_nxdl_root_and_path(nxdl: str):
     -------
     ET.root
         Root element of nxdl file.
+    str
```

Comment:
parameter is called nxdl

Reply:
But this not parameter, but return value, right?

Reply:
or I did get your point.